docs/data/whats-new/WhatsNewLayout.js

```diff
@@ -0,0 +1,243 @@
+        title: 'Better APIs',
+        url: 'https://mui.com/blog/date-pickers-stable-v5/#better-apis-and-improved-customization',
+      },
+      {
+        title: 'Easier customization',
+        url: 'https://mui.com/blog/date-pickers-stable-v5/#better-apis-and-improved-customization',
+      },
+      {
+        title: 'Integrated localization',
+        url: 'https://mui.com/blog/date-pickers-stable-v5/#integrated-localization',
+      },
+    ],
+  },
+  {
+    title: 'Data Grid v5.15',
+    description:
+      'This version brings an amazing set of new supported use cases with the Data Grid Premium.',
+    announcementDate: 'Monday, Aug 12, 2022',
+    url: 'https://mui.com/blog/aggregation-functions/',
+    highlightList: [
+      {
+        title: 'Aggregation in summary rows and row groups',
+        url: 'https://mui.com/blog/aggregation-functions/#wait-what-is-an-aggregation-function',
+      },
+      {
+        title: 'Row pinning',
+        url: 'https://mui.com/blog/aggregation-functions/#row-pinning',
+      },
+    ],
+  },
+  {
+    title: 'New Premium plan',
+    description:
+      'Premium plan announcement, including the most advanced features for data analysis and management.',
+    announcementDate: 'Thursday, May 12, 2022',
+    url: 'https://mui.com/blog/premium-plan-release/',
+    highlightList: [
+      { title: 'Row Grouping', url: '/x/react-data-grid/row-grouping/' },
+      { title: 'Excel export', url: '/x/react-data-grid/export/#exported-rows' },
+    ],
+  },
+  {
+    title: 'MUI X v5.0.0',
+    description: 'A new virtualization engine, and improvements in several APIs.',
+    announcementDate: 'Monday, Nov 22, 2021',
+    url: 'https://mui.com/blog/mui-x-v5/',
+    highlightList: [
+      {
+        title: 'New virtualization engine',
+        url: 'https://mui.com/blog/mui-x-v5/#a-new-virtualization-engine',
+      },
+      {
+        title: 'Improved state management',
+        url: 'https://mui.com/blog/mui-x-v5/#improved-state-management',
+      },
+      {
+        title: 'Simplified style customization',
+        url: 'https://mui.com/blog/mui-x-v5/#simplified-style-customization',
+      },
+    ],
+  },
+];
+
+function BlogCard(props) {
+  return (
+    <Card
+      sx={{
+        height: '100%',
+        display: 'flex',
+        flexDirection: 'column',
+        px: 2,
+        pt: 2,
+        pb: 0,
+        gap: 1.5,
+        borderRadius: '12px',
+        backgroundColor: (theme) =>
+          theme.palette.mode === 'dark'
+            ? 'primaryDark.700'
+            : `${alpha(theme.palette.grey[50], 0.4)}`,
+        borderColor: 'divider',
+      }}
+      component="article"
+      variant="outlined"
+    >
+      <CardContent sx={{ p: 0 }}>
+        <Box
+          sx={{
+            borderBottom: '1px solid',
+            borderColor: 'divider',
+            marginBottom: 1,
+            px: 1,
+          }}
+        >
+          <Typography variant="caption" color="text.secondary">
```

Comment:
I noticed that the `fontFamily` is not in line with the rest of the doc, maybe we could align it? (Using the `BrandingProvider` should be enough I think) 

Font family now: 
![image](https://github.com/mui/mui-x/assets/72460825/0f6aed7e-69a2-44c7-9ffa-646cf4a17a2e)
Standard:
![image](https://github.com/mui/mui-x/assets/72460825/1dd83be1-a777-41af-a465-de55b0f0df05)

Reply:
@noraleonte, I applied the `<BrandingProvider/>` on the entire grid. Let me know what you think.

Reply:
I feel like it's a lot better 💙

Reply:
I applied the `font family` directly and reverted from using the `BrandingProvider` due to failing regression tests (perhaps something related to exports, since using locally didn't trigger the errors).

Let me know what you think.